### PR TITLE
Scrim Locking

### PR DESCRIPTION
**Pull request by bwpikaard**

### CODE REVIEW

**Comment by 0c370t** [comment]
Task linked: [CU-2e6wxew Add "Locked" State to scrim FSM](https://app.clickup.com/t/2e6wxew)

File: clients/web/src/lib/components/organisms/scrims/modals/ScrimManagementModal.svelte [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
-        activeScrims, type ActiveScrims, cancelScrimMutation, type CurrentScrim,
+        activeScrims,
+        type ActiveScrims,
+        cancelScrimMutation,
+        type CurrentScrim,
     } from "$lib/api";
+    import {
+        lockScrimMutation,
+        unlockScrimMutation,
+    } from "../../../../api/mutations/SetScrimLocked.mutation";
```
Comment (gankoji) [inline clients/web/src/lib/components/organisms/scrims/modals/ScrimManagementModal.svelte:12]: $lib/api

File: clients/web/src/routes/management/index.svelte [hunk 2]
```diff
@@ -9,7 +13,7 @@
 
     export const load: Load = ({session}) => {
         if (session.user) {
-            if (session.user.orgTeams.some((s:number) => s === MLEDB_ADMIN)) {
+            if (session.user.orgTeams.some(s => s === MLEDB_ADMIN)) {
```
Comment (gankoji) [inline clients/web/src/routes/management/index.svelte:16]: Why remove the type declaration?

**Review by gankoji — APPROVED** [review]

File: clients/web/src/routes/management/index.svelte (hunk 2 above)
Comment (bwpikaard) [reply clients/web/src/routes/management/index.svelte:16]: It isn't needed since orgTeams is defined as number[], it already knows it's a number.

**Review by gankoji — APPROVED** [review]

**Pull request merged**